project.clj

```diff
@@ -36,7 +41,9 @@
                          [org.apache.maven.wagon/wagon-provider-api "2.10"]
                          [org.apache.commons/commons-exec "1.3"]
                          [org.apache.commons/commons-compress "1.8"]
+                         [org.apache.commons/commons-lang3 "3.4"]
                          [org.apache.httpcomponents/httpclient  "4.5.1"]
```

Comment:
Unrelated to this PR, but while we're here, I wonder if we should bump `org.apache.httpcomponents/httpclient` from '4.5.1' to '4.5.2'.  That's the version that `puppetlabs/http-client "0.7.0"` pulls in.  For the benefit of the users of `puppetlabs/http-client`, I like keeping the versions of the apache http components in sync.   `puppetlabs/http-client` is using version '4.4.5' of `org.apache.httpcomponents/httpcore` already, so I think the line below is fine.